content/blog/deploying-convoy-to-fly.md

```diff
@@ -0,0 +1,210 @@
+
+To create fly Postgres, run the following command on your fly.io authenticated machine:
+
+```
+fly postgres create
+```
+
+what follows is an interactive session where you are asked to:
+
+ 1. Select a name for your Postgres app. An example is `convoy-pg-data`, you will be asked to choose another name if an app on convoy already uses the name you have selected.
+
+ 2. Select an organization. The default organization **personal** is selected automatically if have not created any other organization.
+
+ 3. Select a region where you want Postgres to run.
+
+ 4. Select a plan. Choose a plan according to your current needs. Of course, the development plan should be preferred if you're only testing out things.
+
+ 5. You'd be asked if you want to Scale single node pg to zero after one hour. Again, choose according to your need.
+
+After these, the database will be deployed, copy and save your credentials as the output suggests. In a few moments, you will need to attach this Postgres deployment to your Convoy app.
+
+![output from creating a fly postgres instance](/blog-assets/static-ip-of-an-outbound-request.jpg)
+
+### - Deploy Redis: 
+Similar to fly Postgres, you can automatically create a [Redis](https://fly.io/docs/reference/redis/) database on fly.io without having to manually configure and run a Redis application. But this is done only with the Fly CLI tool, so open up your terminal once again and run the command below to get started:
+
+```
+flyctl redis create
+```
+
+Just as with the Postgres creation terminal session, provide the desired answers to the configuration questions asked:
+
+```
+? Select Organization: fly-apps
+? Choose a primary region (can't be changed later)
+? Would you like to enable eviction?
+? Optionally, choose one or more replica regions (can be changed later)
+? Select an Upstash Redis plan
+```
+
+After your Redis database is ready, your connection URL will be printed out, copy and save this as you would need it to configure convoy later.
+
+### - Deploy Typesense
+
+Typesense is another dependency of Convoy, hence you need to run at least an instance of it before deploying convoy. In this guide you'll run the `typesense/typesense:0.24.0` image of typesense. Unlike the Postgres and Redis databases that you deployed earlier, you will need to create a configuration file named `fly.toml` in order to deploy Typesense. All fly.io apps are configured via a `fly.toml` file, except the fly Postgres and Redis by Upstash databases that are automatically provisioned via the Fly CLI. But this config file also gets automatically generated when you launch an application with the `flyctl launch` command, after which you can manually edit it to fit your configuration needs.
```

Comment:
@Shulammite-Aso Please include that they have a cloud platform you can, so you don't necessarily have to deploy Typesense but for the purpose of this article we will be deploying Typesense.

Reply:
Done